reana_commons/openapi_specifications/reana_workflow_controller.json

```diff
@@ -219,6 +219,13 @@
             "required": true,
             "type": "string"
           },
+          {
+            "description": "Shared root path to store the workflow",
```

Comment:
A better description could be:

- A root path under which the workflow workspaces are stored.

Reply:
Please add yourself to the AUTHORS file.

Please add a release commit on top.